pig/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
that makes sense.  let's go with that.

Reply:
Interesting. These weren't resources in PIG, but the type universe are resources in PartiQL Lang Kotlin. I still think they should be in `src/main/pig` and `src/test/pig` and PartiQL Lang Kotlin will be updated accordingly. PartiQL Lang Kotlin example package structure is also strange because there are both Kotlin and Java sources but the directory structure is still special. I will eventually refactor PartiQL Lang Kotlin to use conventional structure to remove the inconsistencies and extra config